boards/arm/lpcxpresso55s36/lpcxpresso55s36-pinctrl.dtsi

```diff
@@ -57,4 +57,53 @@
 		};
 	};
 
+	pinmux_flexpwm0_pwm0: pinmux_flexpwm0_pwm0 {
+		group0 {
+			pinmux = <PWM0_A0_PIO1_20>,
+				<PWM0_B0_PIO1_17>;
+			slew-rate = "standard";
+		};
+	};
+
+	pinmux_flexpwm0_pwm1: pinmux_flexpwm0_pwm1 {
+		group0 {
+			pinmux = <PWM0_A1_PIO1_6>,
+				<PWM0_B1_PIO1_22>;
+			slew-rate = "standard";
+		};
+	};
```

Comment:
Is there a reason why you don't make a node with 2 subnodes instead of 2 unrelated nodes with this curious naming scheme? I actually don't know, I'm just curious

Reply:
These both represent an individual sub-module on the flexpwm that can be configured independently via their independent nodes, while combining these nodes into one node doesn't necessarily lose the ability to configure the sub modules independently, it certainly makes the idea less clear. The naming scheme comes from the dts files which were inspired by other pwm nodes already in Zephyr.